metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1693,7 +1693,7 @@ def parse_custom_sql(
     ) -> Optional["SqlParsingResult"]:
         database_info = datasource.get(c.DATABASE) or {
             c.NAME: c.UNKNOWN.lower(),
-            c.CONNECTION_TYPE: "databricks",
+            c.CONNECTION_TYPE: datasource.get(c.CONNECTION_TYPE) or "databricks",
```

Comment:
I don't think the default of "databricks" makes sense - that should be removed

Reply:
Just wondering if removal of default as "databricks" don't affect any other thing. Might have some reason behind adding default as "databricks" in below PR:
https://github.com/datahub-project/datahub/pull/9838

Let me know if it still needs to remove.